opendcs-rest-api/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~  Copyright 2024 OpenDCS Consortium and its Contributors
+  ~
+  ~  Licensed under the Apache License, Version 2.0 (the "License")
+  ~  you may not use this file except in compliance with the License.
+  ~  You may obtain a copy of the License at
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~  Unless required by applicable law or agreed to in writing, software
+  ~  distributed under the License is distributed on an "AS IS" BASIS,
+  ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~  See the License for the specific language governing permissions and
+  ~  limitations under the License.
+  -->
+
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://java.sun.com/xml/ns/javaee"
+         xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
+         version="3.0">
+    <display-name>OpenDCS REST API</display-name>
+    <servlet>
+        <servlet-name>jersey-servlet</servlet-name>
+        <servlet-class>org.glassfish.jersey.servlet.ServletContainer</servlet-class>
+        <init-param>
+            <param-name>javax.ws.rs.Application</param-name>
+            <param-value>org.opendcs.odcsapi.res.RestServices</param-value>
+        </init-param>
+        <load-on-startup>1</load-on-startup>
+    </servlet>
+    <session-config>
+        <session-timeout>180</session-timeout> <!-- Timeout after 3 hours of inactivity -->
+    </session-config>
+    <context-param>
+        <param-name>opendcs.rest.api.authorization.type</param-name>
+        <param-value>openid,sso,basic</param-value>
+    </context-param>
+    <context-param>
+        <param-name>opendcs.rest.api.authorization.expiration.duration</param-name>
+        <param-value>PT15M</param-value>
+    </context-param>
+    <context-param>
+        <param-name>opendcs.rest.api.cwms.office</param-name>
+        <param-value>SPK</param-value>
```

Comment:
Do you really want this set to a specific office?

Reply:
No, but if we're being honest with ourselves make it generic is not likely to work at the moment.

For CWMS the schema includes the office code on any root tables (not certain leaf tables) but none of the objects within Java include an "office" field.

There are tentative plans to implement a standard "Organization Field" across implementations. Doesn't quite map to HDB, but given the way I presented it to Andrew he said they could adapt fairly easily.

Reply:
web.xml shouldn't have the SPK default, removed to leave blank